spyderlib/plugins/__init__.py

```diff
@@ -264,6 +270,7 @@ def __init__(self, main=None, **kwds):
         """Bind widget to a QMainWindow instance"""
         super(SpyderPluginMixin, self).__init__(**kwds)
         assert self.CONF_SECTION is not None
+        self.PLUGIN_PATH = Path(inspect.getfile(self.__class__)).parent

```

Comment:
This seems too complex to my taste, so i don't like it too much.

Couldn't we simplify it somehow?